airbyte-integrations/connectors/destination-redshift/src/main/java/io/airbyte/integrations/destination/redshift/RedshiftDestination.java

```diff
@@ -63,13 +60,13 @@ public static boolean isCopy(final JsonNode config) {
     final var emptyRegion = regionNode == null || regionNode.asText().equals("");
 
     if (bucketNode == null && emptyRegion && accessKeyIdNode == null && secretAccessKeyNode == null) {
-      return false;
+      return DestinationType.INSERT_WITH_SUPER_TMP_TYPE;
```

Comment:
I made a similar comment on the BigQuery destination, we should probably introduce a similar warning here to recommend using staging instead of direct insert too!
see https://github.com/airbytehq/airbyte/pull/11776/files#r844957299

Reply:
WARN was added. Thanks for recommendation.